components/src/core/score-card/public-api.ts

```diff
@@ -0,0 +1,2 @@
+export * from './scorecard.component';
+export * from './scorecard.module';
```

Comment:
Question: why does Hero only export the module, while all of the other components export the module and the component?

Reply:
This is actually a bug in the Hero public-api.  I've updated it to export all from `hero.component` and `hero-banner.component` as well.  The reasoning for this is to allow the `HeroComponent` and `HeroBannerComponent` to be propagated upwards so a user can simply `import { HeroComponent } from '@pxblue/angular-components'`.